lightstep/resource_metric_condition.go

```diff
@@ -322,12 +322,10 @@ func getThresholdSchemaMap() map[string]*schema.Schema {
 		"critical": {
 			Type:     schema.TypeString,
 			Optional: true,
-			Default:  "",
```

Comment:
these default values were causing a non-nil thresholds block to be added even when it's empty so I removed them -- they're both optional so that seems fine to me